src/points/slice.ts

```diff
@@ -39,6 +39,7 @@ export interface State {
   pendingPointsEvents: PendingPointsEvent[]
   pointsBalanceStatus: 'idle' | 'loading' | 'error' | 'success'
   pointsBalance: string
+  introHasBeenSeen: boolean
```

Comment:
nit: perhaps `introHasBeenDismissed` is a slightly more accurate name, since you can "see" the intro screen and this can still be `false`. it only becomes `true` once you actually press the dismiss button.